### PR TITLE
Improved offline installation steps

### PR DESCRIPTION
**Pull request by davidcr01**
## Description

Related: https://github.com/wazuh/wazuh-packages/issues/1874
The aim of this PR is to change the offline installation steps of the documentation.

The `./wazuh-install.sh -dw <deb|rpm>` command may lead to confusion. So it is split into two alternatives.
### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by c-bordon — APPROVED** [review]

File: source/deployment-options/offline-installation.rst [hunk 1]
```diff
@@ -22,13 +22,24 @@ Prerequisites
 Download the packages and configuration files
 ---------------------------------------------
 
-#. Replace ``<deb|rpm>`` in the following command with your choice of package format and run it from a Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures.
+#. Run the following commands with your choice and execute them from a Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures.
```
Comment (teddytpc1) [inline source/deployment-options/offline-installation.rst:25]: Suggestion for:
>`Run the following commands with your choice and execute them from a Linux system with Internet connection`

Change it to: 
> `Run the following commands with your choice from a Linux system with Internet connection`

File: source/deployment-options/offline-installation.rst [hunk 2]
```diff
@@ -22,13 +22,24 @@ Prerequisites
...
 
    .. code-block:: console
 
       # curl -sO https://packages.wazuh.com/|WAZUH_CURRENT_MINOR|/wazuh-install.sh
       # chmod 744 wazuh-install.sh
-      # ./wazuh-install.sh -dw <deb|rpm>
+    
+   If your system is RPM-based, run the following command:
```
Comment (teddytpc1) [inline source/deployment-options/offline-installation.rst:32]: We can change this section to use tabs for RPM and Debian options.

**Review by teddytpc1 — CHANGES_REQUESTED** [review]

File: source/deployment-options/offline-installation.rst (hunk 2 above)
Comment (davidcr01) [reply source/deployment-options/offline-installation.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/6686/commits/bae4dcc84a08e98181f4c81dcb125d83cc0c0596.

File: source/deployment-options/offline-installation.rst (hunk 1 above)
Comment (davidcr01) [reply source/deployment-options/offline-installation.rst]: Done in https://github.com/wazuh/wazuh-documentation/pull/6686/commits/bae4dcc84a08e98181f4c81dcb125d83cc0c0596.

**Review by teddytpc1 — APPROVED** [review]

File: source/deployment-options/offline-installation.rst [hunk 3]
```diff
@@ -22,13 +22,25 @@ Prerequisites
 Download the packages and configuration files
 ---------------------------------------------
 
-#. Replace ``<deb|rpm>`` in the following command with your choice of package format and run it from a Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures.
+#. Run the following commands with your choice from a Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures.
```
Comment (javimed) [inline source/deployment-options/offline-installation.rst:25]:
```suggestion
#. Run the following commands from any Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures. Select the package format to download.
```

**Review by javimed — CHANGES_REQUESTED** [review]

**Review by teddytpc1 — APPROVED** [review]

**Review by javimed — APPROVED** [review]

**Pull request merged**